src/test/java/io/zold/api/WalletTest.java

```diff
@@ -45,7 +46,10 @@
  * @checkstyle JavadocMethodCheck (500 lines)
  * @checkstyle JavadocVariableCheck (500 lines)
  * @checkstyle MagicNumberCheck (500 lines)
+ * @checkstyle LineLengthCheck (500 lines)
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
```

Comment:
@Vatavuk 3 lines sufficient here.

Reply:
@carlosmiranda LineLength should stay as is (there are several tests with long strings), I've updated dataAbstraction to 3 lines

Reply:
@Vatavuk yes, I'm only referring to `ClassDataAbstractionCoupling`